python/tank/pipelineconfig.py

```diff
@@ -327,12 +378,27 @@ def _update_metadata(self, updates):
         self._pc_id = curr_settings.get("pc_id")
         self._pc_name = curr_settings.get("pc_name")
 
+    def get_pipeline_config_file_location(self):
+        """
+        Returns the location of the pipeline_configuration.yml file.
+        """
+        return os.path.join(
+            self._pc_root, "config", "core",
+            constants.PIPELINECONFIG_FILE
+        )
+
+    def get_yaml_cache_location(self):
```

Comment:
can this be private? why are we adding it to the interface? do we really need a method? It looks like it's just called from one place.